stack/adding-a-new-package.rst

```diff
@@ -53,37 +53,49 @@ All LSST DM repositories on GitHub must be configured by a repository administra
 
-In the image above no automated status checks are being performed because we have not yet enabled any.
-GitHub requires that at least one check runs before the up-to-date checks are enabled so a GitHub Action or Travis job **must** be provided if the GitHub merge button is to be used.
-GitHub Actions or Travis do not replace normal testing done with a :doc:`Jenkins job <jenkins-stack-os-matrix>`.
+.. note::
+
+  In the image above, status checks have been added for the ``lint`` GitHub action, however, the ``lint`` GitHub action may not be available when the package is first created.
+  To make the ``lint`` action an available option in this dropdown menu, you first need to generate a simple test pull request job.
+  When the pull request has been submitted, the status of the action may be checked in the ``Actions`` tab.
+  Once the action has completed, you can return to the branch settings page on GitHub.
+  Now you will see that the ``main`` branch is listed along with an EDIT button.
+  In the ``Branch protection rule`` settings, type ``lint`` into the search box and select the ``lint`` GitHub action.
+  Enable the ``lint`` action and save.
+
+GitHub requires that at least one check runs before the up-to-date checks are enabled, so a GitHub Action **must** be provided if the GitHub merge button is to be used.
+GitHub Actions do not replace normal testing done with a :doc:`Jenkins job <jenkins-stack-os-matrix>`.
 For packages that contain Python, it is useful to add a simple GitHub Action by selecting "Actions" from the GitHub repository page, selecting "New Workflow" if necessary, and choosing the "LSST DM Python lint Workflow".
 If Python typing is used, it can be checked using ``mypy`` via the "LSST DM Python mypy Workflow".
 Similarly, YAML files can be checked via the "LSST DM YAML lint Workflow", and shell scripts can be checked via the "LSST DM shellcheck Workflow".
-(Both of those checks can be configured, either via an external file such as ``.yamllint.yaml`` or via modifications to the workflow as described in the link in the shellcheck workflow.)
-If nothing seems appropriate the "LSST DM null Workflow" should be enabled to allow GitHub to do the checks it needs.
-
-Pull requests will automatically run the GitHub Actions and their results will be visible in the "Checks" tab of the pull request on GitHub.
-
-When the first job completes you can return to the branches settings page on GitHub.
-Now you will see that the ``main`` branch is listed along with an EDIT button.
-The branch protection rules will now list the GitHub Actions checks in the "up to date before merging" section.
-Enable these and save.
-Your branch protections screen should then look something like this:
+(All of these checks can be configured, either via an external file such as ``lint.yaml``, or via modifications to the workflow as described in the link in the shellcheck workflow.)
```

Comment:
`lint.yaml` is not actually a configuration file for any of the utilities.  `.yamllint.yaml` in the repo root directory really is a configuration for the YAML linter.